test/errors/test_FPDF_errors.py

```diff
@@ -105,3 +106,15 @@ def test_repeated_calls_to_output(tmp_path):
     pdf = fpdf.FPDF()
     assert_pdf_equal(pdf, HERE / "repeated_calls_to_output.pdf", tmp_path)
     assert_pdf_equal(pdf, HERE / "repeated_calls_to_output.pdf", tmp_path)
+
+
+def test_unsupported_image_filter_error():
+    class Object:
+        pass
+
+    img = Object()
```

Comment:
Why building a mock image here, and not simply passing a real `PIL.Image`?

Reply:
I tried to write minimal code to raise the exception. But yes it's better to test other things at the same time